packages/mdl-checkbox/mdl-checkbox.scss

```diff
@@ -18,9 +18,6 @@
 @import "./variables";
 @import "./keyframes";
 
-// TODO: Docs
-// TODO: x-browser test!
-
 @function mdl-checkbox-transition($property, $timing-function, $delay: 0ms, $duration: $mdl-checkbox-transition-duration) {
```

Comment:
Ah yeah I figured since I was looking at that file anyway I'd take the time to remove some outdated TODOs. I can update the commit msg and/or remove those changes if you want. Let me know!